spring-cloud-data-rest-resource/src/main/java/org/springframework/cloud/data/rest/resource/TaskDefinitionResource.java

```diff
@@ -30,6 +31,12 @@
 
 	private String dslText;
 
+	/**
+	 * Stream status (i.e. running, complete, etc).
+	 */
+	private String status = "";
```

Comment:
+1 for this never being null or empty.  Even though we're not using an enum, we should still use something.  In Spring Batch, we initialize  `JobExecution#status` to `BatchStatus#STARTING` since it's the first possible state...